repos/system_upgrade/el8toel9/actors/firewalldcollectusedobjectnames/libraries/private_firewalldcollectusedobjectnames.py

```diff
@@ -0,0 +1,92 @@
+from leapp.models import FirewalldUsedObjectNames
+
+try:
+    from firewall.core.fw import Firewall
+except ImportError:
+    pass
+
+
+def is_zone_in_use(conf):
+    # This does not account for zone assignments given by NetworkManager.
+    if conf.get('interfaces', []) or conf.get('sources', []):
+        return True
+
+    return False
+
+
+def is_policy_in_use(conf, used_zones):
+    # A policy is in use if both ingress_zones and egress_zones contain at
+    # least one of following: an active zone, 'ANY', 'HOST'.
+    for zone in conf.get('ingress_zones', []):
+        if zone in ['ANY', 'HOST'] or \
+           zone in used_zones:
+            break
+    else:
+        return False
+    for zone in conf.get('egress_zones', []):
+        if zone in ['ANY', 'HOST'] or \
+           zone in used_zones:
```

Comment:
Nit: this line should easily fit to the previous one. Same for the line 22. However, this is probably matter of personal taste. It just looks like indentation is broken on first look.

Reply:
ACK. I'll fix it when I rebase. It's probably an artifact from code refactoring during development (i.e. original was indented more).